README.md

```diff
@@ -65,63 +66,72 @@ require 'coinbase'
 
 ### Initialization
 
-The SDK requires a Base Sepolia RPC node, and uses the public instance (https://sepolia.base.org) by default.
-This instance is rate-limited, but you can also provision your own on the
-[Coinbase Developer Platform](https://portal.cloud.coinbase.com/products/base).
-
-
-You can configure the SDK with your Base Sepolia RPC node by copying your node URL from the CDP portal
-and setting the following:
+To start, [create a CDP API Key](https://portal.cdp.coinbase.com/access/api). Then, initialize the Platform SDK by passing your API Key name and API Key private key via the `configure` method:
 
 ```ruby
 Coinbase.configure do |config|
-    config.base_sepolia_rpc_url = 'https://api.developer.coinbase.com/rpc/v1/base/your-node-id'
+    config.api_key_name = 'organizations/your-org-id/apiKeys/your-api-key-id'
+    config.api_key_private_key = '-----BEGIN EC PRIVATE KEY-----\nyour-api-key-private-key\n-----END EC PRIVATE KEY-----\n'
 end
 ```
 
-### Wallets and Addresses
+This will allow you to authenticate with the Platform APIs and get access to the `default_user`.
+
+```ruby
+u = Coinbase.default_user
+```
 
-A Wallet is a collection of Addresses on the Base Sepolia Network, which can be used to send and receive crypto.
+### Wallets, Addresses, and Transfers
```

Comment:
add balances?

Reply:
Per offline convo, this can be just in the CDP docs